base_export_manager/models/ir_exports_line.py

```diff
@@ -53,104 +62,106 @@ def _default_model1_id(self):
         """Default model depending on context."""
         return self.env.context.get("default_model1_id", False)
 
-    @api.multi
-    @api.depends("field1_id", "field2_id", "field3_id")
+    @api.one
```

Comment:
Isn't best to leave api.maulti here, since we have confirmation that Odoo intends to deprecated api.one ?